includes/class-wc-admin-reports-sync.php

```diff
@@ -571,6 +572,7 @@ public static function get_user_ids_for_batch( $days, $skip_existing, $query_arg
 		}
 
 		if ( $skip_existing ) {
+			$query_args['paged'] = 1;
```

Comment:
What do you think about moving this to https://github.com/woocommerce/woocommerce-admin/blob/fc20a2f2b18b44e9d87e139e5127874b9ac0da43/includes/class-wc-admin-reports-sync.php#L664 ?

Reply:
Sounds good, changed in 93ebea7f92d615b9d025f019850207ab4abc9c39. I also added an explanatory comment.